README.md

```diff
@@ -78,17 +74,17 @@ After you're finished please remove all the comments and instructions!
 
 - HTML
 - CSS
+- Javascript
 - Visual Studio Code 
-- Figma
+- Behance
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-- [Live Demo Link](https://joramot.github.io/capstone_one_global_summit/)
+- [Live Demo Link]( Comming soon!!!)
```

Comment:
- [ ] It is required to deploy your web pages with GitHub Pages. Kindly fulfill that requirement